upi/azure/05_masters.json

```
@@ -32,6 +32,7 @@
     },
     "privateDNSZoneName" : {
       "type" : "string",
+      "defaultValue" : "",
       "metadata" : {
         "description" : "Name of the private DNS zone the master nodes are going to be attached to"
```

Change the description to indicate that this field is not used.
```suggestion
        "description" : "unused"
```